src/main/java/org/apache/commons/csv/Lexer.java

```diff
@@ -22,6 +22,8 @@
 import java.io.Closeable;
 import java.io.IOException;
 
+import org.apache.commons.io.IOUtils;
```

Comment:
We do not require this import for the code. You can use it directly in the javadoc.